scss/_objects.grid.scss

```diff
@@ -4,47 +4,63 @@
     Grid system objects, works together with utility width classes
 */
 
+
+// Max width container
 .o-container {
     margin-left: auto;
     margin-right: auto;
     max-width: $grid-max-width;
 }
 
-// No gutters
 
 .o-grid {
-    @include clearfix;
+    display: flex;
+    align-items: stretch;
+    flex-direction: row;
+    flex-wrap: wrap;
 }
 
-// Default gutter
+    // 100% width by default
+    .o-grid__item {
+        width: 100%;
```

Comment:
I use `width: 100%` to make sure it is compatible with the width utility classes. In my tests it worked fine and because we don't really change the logic of the grid system I would keep this.